api/models.py

```diff
@@ -12,7 +12,7 @@ class UserProfile(models.Model):
 	)
 
 	user = models.OneToOneField(User) 
-	#id = models.AutoField(primary_key=True) 
+	id = models.AutoField(primary_key=True) 
```

Comment:
I have seen this somewhere that id is generated automatically. So if I delete this id line in my model, how should in write my view with id? I tried some combinations with id instead of githubuser, but nothing worked. Thx :)

def member_page(request):
	user = UserProfile.objects.get(githubuser="PettraV")
	return render(request, 'pages/member_page.html',{'user': user})